src/arrayMethodReduce.js

```diff
@@ -5,7 +5,15 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let firstValue = initialValue;
```

Comment:
```suggestion
    let result = initialValue;
```
this is not an initial value